meeting/meeting_november_10_2020.md

```diff
@@ -0,0 +1,158 @@
+  * Done
+    * I have successfully executed such a DLC on an 8-digit (base 10) oracle outcome space in bitcoin-s!
+    * Specifies mult-signature event adaptor points using signature addition, which is compatible with fraud proofs
+    * Specifies CET set compression algorithm and provides analysis yielding simple function for computing the number of CETs
+    * Specifies support for arbitrary payout curves
+    * Specifies precision ranges which allow rounding in order to further compress CET set and reduce computation
+    * Specifies CET signature construction and validation
+  * TODO
+    * Add support for signed (`+-`) numeric outcomes
+      * I am currently leaning towards running the unsigned algorithm on positive and negative numbers separately so as to keep things simple at very little cost, any objections?
+      * **#Goal(Approve this approach or decide on alternative)**
+    * Integration with other spec files
+    * Pretty diagrams
+    * Explicitly recommend base 2 signing for oracles
+    * Scala code -> Python
+    * Separate CET compression spec from payout curve spec (multiple files)
+    * Test Vectors (in future PR)
+  * **#Discussion**
+
+## Non-Interactive Protocol
+
+**#Status_Update**
+
+* The initial specification for on-chain handling has been [merged](https://github.com/discreetlogcontracts/dlcspecs/pull/87)!
+  * NADAV, WRITE THIS SECTION ONCE THINGS ARE MERGED
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* Initial test vectors passed by 3 independent implementations and [merged](https://github.com/discreetlogcontracts/dlcspecs/pull/100)!
+* Dust specification
+  * We currently just hard-code `1000 satoshi` limit
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/11
+* Ordering of inputs and outputs
+  * We plan to use the `serial_id` construction [here](https://github.com/niftynei/lightning-rfc/blob/1aa5dbe2987f0a0e79fa1d04718d91c6d01e07b2/02-peer-protocol.md#the-tx_add_output-message)
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/18
+* Adaptor Signature Specification (assigned to Lloyd)
+  * High level section (motivation, abstract) are higher priority because people have been asking for resources and there aren't any
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/50
+  * **#Query(Lloyd)**
+    * A couple months ago Lloyd [mentioned](https://github.com/discreetlogcontracts/dlcspecs/issues/50#issuecomment-693101689) his anticipated approach, any updates?
+* Adding TLV Streams for extensibility
+  * Does this even make sense given the way we are using TLVs?
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/73
+* Pretty Pictures Wanted!
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/77
+  * **#Query(all)**
+    * Anybody interested or know someone who might be?
+    * Is there anything we can do to get this to happen? Maybe post to telegram or tweet?
+* SIGHASH discussion seems to have gone stale
```

Comment:
For this one, we should consider it for 0.2, for now I expect to be too much complexity.